### PR TITLE
Feature/481 party receipt

### PR DESCRIPTION
**Pull request by mjtravers**
[#481](https://github.com/fecgov/fecfile-web-app/issues/481)

### CODE REVIEW

File: schema/PARTY_RECEIPT.json [hunk 1]
```diff
@@ -0,0 +1,457 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/PARTY_RECEIPT.json",
+    "version": "v0.0.0.0",
+    "title": "PARTY RECEIPT",
+    "description": "Transfer (12)",
```
Comment (toddlees) [inline schema/PARTY_RECEIPT.json:6]: `"description": "Party Receipt (11b)",`

**Review by toddlees — CHANGES_REQUESTED** [review]

File: schema/PARTY_RECEIPT.json [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+        "contributor_fec_id",
+        "contributor_street_1",
+        "contributor_city",
+        "contributor_state",
+        "contributor_zip",
+        "contribution_date",
+        "contribution_amount",
+        "contribution_aggregate",
+        "aggregation_group",
+        "donor_committee_fec_id"
```
Comment (toddlees) [inline schema/PARTY_RECEIPT.json:24]: `"donor_committee_name"`

**Review by toddlees — CHANGES_REQUESTED** [review]

File: schema/PARTY_RECEIPT.json (hunk 1 above)
Comment (mjtravers) [reply schema/PARTY_RECEIPT.json]: fixed

File: schema/PARTY_RECEIPT.json (hunk 2 above)
Comment (mjtravers) [reply schema/PARTY_RECEIPT.json]: fixed. good catch!

**Review by toddlees — APPROVED** [review]

**Review by Elaine-Krauss-TCG — APPROVED** [review]

**Pull request merged**